cypress/support/step-definitions/carousel-steps.js

```diff
@@ -74,6 +86,10 @@ Then('next button is not visible', () => {
   nextArrowButton().should('not.exist');
 });
 
+Then('next button is disabled', () => {
+  nextArrowButton().should('be.disabled');
```

Comment:
needs to be added .and('have.attr', 'disabled')

Reply:
Added 👍